fedimint-api/src/core/server.rs

```diff
@@ -255,11 +282,12 @@ where
         dbtx: &mut DatabaseTransaction<'_>,
         verification_cache: &DynVerificationCache,
         input: &DynInput,
+        module_instance_id: ModuleInstanceId,
```

Comment:
Same here. The fedimint core code should do it.

Reply:
Removed